src/lib/y2storage/clients/inst_disk_proposal.rb

```diff
@@ -97,7 +97,7 @@ def guided_setup(settings)
       end
 
       def expert_partitioner
-        ExpertPartitioner::MainDialog.new.run
+        Y2Partitioner::Clients::Main.run
```

Comment:
(BTW that check would have to be placed at the require above, that would fail earlier...)